### PR TITLE
Further cleanups of the SVG GridOut code.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

**Comment by bangerth** [comment]
That code was just written in individual components of tensors when it could have been done as tensor arithmetic. All tests pass for me.

File: source/grid/grid_out.cc [hunk 1]
```diff
@@ -1529,54 +1529,26 @@ namespace
...
-    projection_decomposition[1] += (projection[1] - camera_position[1] -
-                                    camera_focus * camera_direction[1]) *
-                                   camera_vertical[1];
-    projection_decomposition[1] += (projection[2] - camera_position[2] -
-                                    camera_focus * camera_direction[2]) *
-                                   camera_vertical[2];
-
-    return projection_decomposition;
+    Tensor<1, 3> camera_vertical;
+    cross_product(camera_vertical, camera_horizontal, camera_direction);
```
Comment (masterleinad) [inline source/grid/grid_out.cc]:
```suggestion
    const Tensor<1, 3> camera_vertical = cross_product_3d(camera_horizontal, camera_direction);
```
AFAICT, the version you were using is deprecated.

File: source/grid/grid_out.cc [hunk 2]
```diff
@@ -1529,54 +1529,26 @@ namespace
...
-                                  camera_vertical[0];
-    projection_decomposition[1] += (projection[1] - camera_position[1] -
-                                    camera_focus * camera_direction[1]) *
-                                   camera_vertical[1];
-    projection_decomposition[1] += (projection[2] - camera_position[2] -
-                                    camera_focus * camera_direction[2]) *
-                                   camera_vertical[2];
-
-    return projection_decomposition;
+    Tensor<1, 3> camera_vertical;
```
Comment (masterleinad) [inline source/grid/grid_out.cc]:
```suggestion
```

File: source/grid/grid_out.cc (hunk 1 above)
Comment (bangerth) [reply source/grid/grid_out.cc]: Ah, that's why -- I tried the two argument `cross_product` function, but that didn't work.

**Comment by bangerth** [comment]
Thanks, @masterleinad -- updated.

**Comment by drwells** [comment]
@bangerth greetings from DEN. Please run clang-format.

**Comment by bangerth** [comment]
Krkhrkhrkhr, indentation.... Fixed!

**Comment by masterleinad** [comment]
Thanks!

**Pull request merged**